### PR TITLE
Sending Data about cli uploading annotations

### PR DESCRIPTION
**Pull request by savish28**
Related Issue: https://github.com/Cloud-CV/EvalAI/issues/3312
Related Pull: Request: https://github.com/Cloud-CV/EvalAI/pull/3322

Sends Data about Cli uploading annotations to update `annotations_uploaded_using_cli` to avoid overwriting using Github API.

Relevant Change is on Line 224. (Others are pre-commit changes)

### CODE REVIEW

**Comment by savish28** [comment]
@Ram81 Ping.

File: evalai/utils/common.py [hunk 1]
```diff
@@ -201,20 +220,21 @@ def upload_file_using_presigned_url(challenge_phase_pk, file, file_type, submiss
 
         data = {
             "parts": json.dumps(response.get("parts")),
-            "upload_id": upload_id
+            "upload_id": upload_id,
+            "annotations_uploaded_using_cli": True,
```
Comment (RishabhJain2018) [inline evalai/utils/common.py:224]: Can we please add this in data dict `"annotations_uploaded_using_cli": True,` only when annotations are uploaded?

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: evalai/utils/common.py (hunk 1 above)
Comment (savish28) [reply evalai/utils/common.py]: @RishabhJain2018 Updated the code.

**Review by RishabhJain2018 — APPROVED** [review]

**Comment by savish28** [comment]
@RishabhJain2018 Ping!

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**